content/Foundation/Summit/2023/home.pml

```diff
@@ -94,8 +95,18 @@ Minneapolis, USA
     <li>Networking with other seL4 experts and enthusiasts</li>
     <li>Reception and Dinner</li>
   </ul>
+</div-->
+
+<div class="summit_div">
+  <h3 id="summit-abstracts">
+    Videos, Slides and Abstracts
```

Comment:
I would change the title to "Program, Abstracts, Slides and Videos".
And do the same for the 2022 page.